Configuration/PyReleaseValidation/python/relval_standard.py

```diff
@@ -433,87 +433,101 @@
-workflows[140.065521] = ['Run3-2023_JetMET2022D_RecoHCALOnlyCPU',['RunJetMET2022D','HLTDR3_2023','RECODR3_reHLT_HCALOnlyCPU','HARVESTRUN3_HCALOnly']]
-
-workflows[140.071] = ['',['RunMuon2022E','HLTDR3_2023','RECONANORUN3_reHLT','HARVESTRUN3']]
-workflows[140.072] = ['',['RunZeroBias2022E','HLTDR3_2023','RECONANORUN3_reHLT','HARVESTRUN3']]
-workflows[140.073] = ['',['RunBTagMu2022E','HLTDR3_2023','RECONANORUN3_reHLT','HARVESTRUN3']]
-workflows[140.074] = ['',['RunJetMET2022E','HLTDR3_2023','RECONANORUN3_reHLT','HARVESTRUN3']]
-workflows[140.075] = ['',['RunDisplacedJet2022E','HLTDR3_2023','RECONANORUN3_reHLT','HARVESTRUN3']]
-workflows[140.076] = ['',['RunEGamma2022E','HLTDR3_2023','RECONANORUN3_reHLT','HARVESTRUN3']]
-workflows[140.077] = ['',['RunTau2022E','HLTDR3_2023','RECONANORUN3_reHLT','HARVESTRUN3']]
-workflows[140.078] = ['',['RunMuonEG2022E','HLTDR3_2023','RECONANORUN3_reHLT','HARVESTRUN3']]
-
-### run3 (2023) skims ###
-workflows[140.104] = ['',['RunZeroBias2022D','HLTDR3_2023','RECONANORUN3_reHLT','SKIMZEROBIASRUN3_reHLT']]
-workflows[140.105] = ['',['RunBTagMu2022D','HLTDR3_2023','RECONANORUN3_reHLT','SKIMBTAGMURUN3_reHLT']]
-workflows[140.106] = ['',['RunJetMET2022D','HLTDR3_2023','RECONANORUN3_reHLT','SKIMJETMET0RUN3_reHLT']]
-workflows[140.107] = ['',['RunDisplacedJet2022D','HLTDR3_2023','RECONANORUN3_reHLT','SKIMDISPLACEDJETRUN3_reHLT']]
-workflows[140.109] = ['',['RunEGamma2022D','HLTDR3_2023','RECONANORUN3_reHLT','SKIMEGAMMA0RUN3_reHLT']]
-workflows[140.110] = ['',['RunTau2022D','HLTDR3_2023','RECONANORUN3_reHLT','SKIMTAURUN3_reHLT']]
-workflows[140.111] = ['',['RunMuon2022D','HLTDR3_2023','RECONANORUN3_reHLT','SKIMMUON0RUN3_reHLT']]
-workflows[140.112] = ['',['RunMuonEG2022D','HLTDR3_2023','RECONANORUN3_reHLT','SKIMMUONEGRUN3_reHLT']]
-workflows[140.113] = ['',['RunNoBPTX2022D','HLTDR3_2023','RECONANORUN3_reHLT','SKIMNOBPTXRUN3_reHLT']]
-workflows[140.114] = ['',['RunHcalNZS2022D','HLTDR3_2023','RECONANORUN3_reHLT','SKIMHCALNZSRUN3_reHLT']]
-workflows[140.115] = ['',['RunHLTPhysics2022D','HLTDR3_2023','RECONANORUN3_reHLT','SKIMHLTPHYSICSRUN3_reHLT']]
-workflows[140.116] = ['',['RunCommissioning2022D','HLTDR3_2023','RECONANORUN3_reHLT','SKIMCOMMISSIONINGRUN3_reHLT']]
-workflows[140.117] = ['',['RunCosmics2022D','HLTDR3_2023','RECOCOSMRUN3_reHLT','SKIMCOSMICSRUN3_reHLT']]
-#workflows[140.118] = ['',['RunParkingBPH2022B','HLTDR3_2023','RECONANORUN3_reHLT','SKIMPARKINGBPHRUN3_reHLT']]
-
+workflows[140.065505] = ['Run3-2023_JetMET2022D_RecoPixelOnlyTripletsCPU',['RunJetMET2022D','HLTDR3_2022','RECODR3_reHLT_Patatrack_PixelOnlyTripletsCPU','HARVESTRUN3_pixelTrackingOnly']]
+workflows[140.065511] = ['Run3-2023_JetMET2022D_RecoECALOnlyCPU',['RunJetMET2022D','HLTDR3_2022','RECODR3_reHLT_ECALOnlyCPU','HARVESTRUN3_ECALOnly']]
+workflows[140.065521] = ['Run3-2023_JetMET2022D_RecoHCALOnlyCPU',['RunJetMET2022D','HLTDR3_2022','RECODR3_reHLT_HCALOnlyCPU','HARVESTRUN3_HCALOnly']]
```

Comment:
I think this will put these 3 wfs out-of-sync (in terms of HLT menu) wrt to their GPU counterparts in
https://github.com/cms-sw/cmssw/blob/905f11684afdfe6fd4e28b49b021ee8f4cab01ca/Configuration/PyReleaseValidation/python/relval_gpu.py#L71
I think the idea would be to update these 6 wfs (3 for CPU and 3 for GPU) to use 2023 data, and 2023 HLT menu (and I guess they should then be numbered `141.*`). It would be best if this could be done already in this PR. Otherwise, I can try to do it in a follow-up.

Reply:
Updated in the next commit, thanks.